### PR TITLE
Add compc and vgroup rollups to tree total and node rows

### PR DESCRIPTION
**Pull request by beckerg**
Signed-off-by: Greg Becker <gbecker@micron.com>

### CODE REVIEW

File: tools/scripts/cn_metrics.py [hunk 1]
```diff
@@ -94,14 +94,14 @@ def full_tree(ybuf: Optional[OrderedDict[str, Any]], opt):
...
+            line = f"k {loc['level']},{loc['offset']},{index:<2}"
 
             col = 0;
             for key, val in kvset.items():
                 width = widthv[col]
                 col += 1
-                print(f"{key} {val:<{width}} ", end="")
-            print()
+                line += f" {key} {val:<{width}}"
+            print(line)
```
Comment (beckerg) [inline tools/scripts/cn_metrics.py:104]: The say join() is much faster than +=, but if I write something like   line = line.join(f" {key} {val}")   then it seems to go into an infinite loop eating up all the memory on the system (yes, all 512G).

Comment (tristan957) [reply tools/scripts/cn_metrics.py:104]: I think you have the arguments reversed for .join() :)

Comment (beckerg) [reply tools/scripts/cn_metrics.py:104]: wait, wouldn't that render the line in the reverse order i want???

Comment (alexttx) [reply tools/scripts/cn_metrics.py:104]: Make a list of text segments and do one join at the end when you print.

Comment (beckerg) [reply tools/scripts/cn_metrics.py:104]: Cool, that works.  But it doesn't seem any faster than line += ...   Maybe if I had much larger data set...

**Review by tristan957 — APPROVED** [review]
You keep making my life harder by touching the rest code! :)

**Comment by beckerg** [comment]
Sorry!  Alex made me do it..

**Review by alexttx — APPROVED** [review]

**Comment by beckerg** [comment]
FWIW:  It takes about 75ms for curl to request the tree, kvdb_rest to generate the yaml, and curl to read it all off the wire.  For ycsb this is about 416KB of data, one third of which is white space, and about one third is repeated schema field names.

It then take cn_metrics.py yaml about 3.6 seconds to parse the yaml and load it into a Dict, and roughly 50ms to run through the Dict and output the data for human consumption.

Just ouch...  I can only imagine the CPU is on fire during those 3.6 seconds...

**Pull request merged**